python/common/org/python/types/Function.java

```diff
@@ -19,6 +19,14 @@
     java.util.Map<java.lang.String, org.python.Object> default_kwargs;
     org.python.types.Closure closure;
 
+    public static java.lang.String protect_reserved_words(java.lang.String arg) {
```

Comment:
Oversight. It was written to make the arguments in org.Python exactly match the official standards - but "default" is a Java keyword, so you can't name an argument "default". However, it turns out this conversion wasn't needed due to the way Java handles argument rollout. I'll delete this method.